backends/gstreamer/audio_decoder.rs

```diff
@@ -220,7 +218,7 @@ impl AudioDecoder for GStreamerAudioDecoder {
                                     })?;
 
                                     for position in positions.iter() {
-                                        let buffer = buffer.clone();
+                                        let buffer = buffer.to_owned();
```

Comment:
Why is this necessary, a whole buffer copy instead of just getting a new reference?

Reply:
There's `Sample::get_buffer_owned()` if that's the issue here, which allows to do this without a copy of the buffer. See https://gitlab.freedesktop.org/gstreamer/gstreamer-rs/issues/204 for the background.